atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -173,6 +173,15 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji task ID: %s", self.koji_task_id)
 
+        try:
+            metadata = get_build_json().get("metadata", {})
+            self.original_koji_task_id = int(metadata['labels']['original-koji-task-id'])
+        except Exception:
```

Comment:
What other exceptions are we expecting here other than TypeError or KeyError? Could we be explicit on those here?

Reply:
using the same as in getting koji-task-id above, we don't really care that much in this case how it fails, we either get value or not